tatanka/cmd/demo/main.go

```diff
@@ -243,8 +246,36 @@ func mainErr() (err error) {
 		return errors.New("timed out waiting for SendTankagram to return")
 	}
 
+	// Fiat rate live test requires internet connection.
+	fmt.Println("Testing fiat rates...")
+
+	for _, assetID := range supportedDEXAssetIDs {
+		if err := cl1.SubscribeToFiatRates(assetID); err != nil {
+			return err
+		}
+	}
+
+	// Wait for rate messages.
+	<-time.After(9 * time.Minute) // rates are broadcasted every 8min, wait for 9min.
```

Comment:
Thats a long time to wait for a test app. Also this blocks shutdown I guess?

Reply:
> Can the server just broadcast more? Would it hurt if it was like every 10 seconds?

That would be the case if we had 2 or more reliable websocket feeds for fiat rates cuz we can send out updates as soon as we receive them. But because we are consuming HTTP APIs, broadcasting several times before the next refresh only benefits those who subscribe after the first broadcast but before a second broadcast. The rate hasn't changed by then.

Reply:
> Would be cool if clients could ask what the last broadcast was from someone...

Errm, they can only use that information to confirm if thier current price is up to date? not really useful to them I think.

Reply:
> Errm, they can only use that information to confirm if thier current price is up to date? not really useful to them I think.

Maybe I am misunderstanding, but the broadcasts can only be caught live correct? What about a person that comes online shortly after the broadcast? They must wait 8 minutes or so correct?

```
    ~8 minutes between
<--|------------------|--x---------------|------------------|------------------|-->
   ^ broadcast           ^ what about a guy that connects here?
```

Reply:
> They must wait 8 minutes or so correct?

Yes, this is now configurable, and it can be every 4/6/8/10 min.

Reply:
Why not 10 seconds? Sorry to be pedantic about this.

Reply:
It's no longer upto the server to decide, whenever we get rates, users will receive them immediately. Some might have to wait until the next rate update.

Reply:
Resolved in https://github.com/decred/dcrdex/pull/2736/commits/1751a1634c4cb79c63269d6953c53fcc5e0db363